lib/pbench/test/unit/server/test_schema.py

```diff
@@ -1,19 +1,67 @@
+from http import HTTPStatus
 import dateutil
 import pytest
 from typing import Callable
```

Comment:
These aren't in order.  (I don't know what the order _should_ be, but I can tell it ain't this.)

Reply:
I'm not sure; if `http` is built-in, then it's in order, although we'd normally have a blank. I think I let vscode add this automatically, and while it clearly has a strong notion of ordering it doesn't always conform to our notion of ordering.

Reply:
`http` and `typing` each appear to be part of the standard library; `dateutil` and `pytest` are not.  So, the latter should not be interposed between the former, per our current methodology.  😝